compiler/rustc_target/src/spec/uefi_msvc_base.rs

```diff
@@ -47,7 +47,7 @@ pub fn opts() -> TargetOptions {
         stack_probes: StackProbeType::Call,
         singlethread: true,
         linker: Some("rust-lld".into()),
-        relocation_model: RelocModel::Static,
+        relocation_model: RelocModel::Pic,
```

Comment:
`Pic` is the default mode so this line can be just removed.

Reply:
done